services/QuillLMS/client/app/bundles/Staff/helpers/evidence/miscHelpers.tsx

```diff
@@ -411,3 +419,57 @@ export const activitySessionIndexResponseHeaders = [
     Cell: props => props.value
   },
 ]
+
+export function colorCodeAttemptsCount(attemptCount, isOptimal) {
+  if(isOptimal) {
+    return attemptCount
+  }
+  return <p className="sub-optimal-attempt">{attemptCount}</p>
+}
+
+export function formatSessionsData(activityId, activitySessions: ActivitySessionInterface[]) {
+  return activitySessions.map(session => {
+    const { start_date, session_uid, because_attempts, because_optimal, but_attempts, but_optimal, so_attempts, so_optimal, complete } = session;
+    const dateObject = new Date(start_date);
+    const date = moment(dateObject).format("MM/DD/YY");
+    const time = moment(dateObject).format("hh:mm a");
```

Comment:
I think you can just do `moment(dateObject).format("MM/DD/YY hh:mm a")` here unless you need them to be two separate variables (it looks like you don't, below).

Reply:
good catch, I'll update